assets/sass/v2/_identify.scss

```diff
@@ -33,16 +34,19 @@
       margin-bottom: 0.83em;
     }
     .button {
```

Comment:
the selectors here are weird here I guess we should get rid of sign-in-with-idp and custom-buttons. cc @jhe-okta 

https://github.com/okta/okta-signin-widget/pull/2083/files#diff-0e0a6554c5f0823c8b023835c2ba22e6fd6a66496d4534cc46c29a88943c3824R30

why would we have a okta-verify-container inside the sign-in-with-idp div.

Reply:
I don't remember the context. But usually, it is because the change only applies to a certain UI.

Reply:
@jasleennuno-okta  just realized that https://github.com/okta/okta-signin-widget/pull/2083/files#diff-0e0a6554c5f0823c8b023835c2ba22e6fd6a66496d4534cc46c29a88943c3824R44 

needs those classes at the top to be encapsulated. Ignore my comment.